app/src/main/java/org/fossasia/openevent/app/data/models/Ticket.java

```diff
@@ -19,7 +20,7 @@
     private long quantity;
     private String type;
 
-    @ForeignKey(stubbedRelationship = true)
+    @ForeignKey(stubbedRelationship = true, onDelete = ForeignKeyAction.CASCADE)
```

Comment:
The onDelete trigger in the Event itself will need to be changed, but I have already set it to CASCADE. There is a bug however which does not cascade deletion in the library itself

Now, it won't create any problem because these are linked with the ID of event, so won't be loaded when event is deleted

So, deleting the event only should be enough